examples/meta/src/neural_nets/feedforward_net_classification.sg

```diff
@@ -0,0 +1,45 @@
+CSVFile f_feats_train("../../data/fm_train_real.dat")
+CSVFile f_feats_test("../../data/fm_test_real.dat")
+CSVFile f_labels_train("../../data/label_train_twoclass.dat")
+
+#![create_features]
+RealFeatures features_train(f_feats_train)
+RealFeatures features_test(f_feats_test)
+BinaryLabels labels_train(f_labels_train)
+#![create_features]
+
+#![add_layers]
+int num_feats = features_train.get_num_features()
+NeuralInputLayer input_layer(num_feats)
+NeuralLinearLayer hidden_layer(50)
+NeuralSoftmaxLayer output_layer(2)
+DynamicObjectArray layers()
+layers.append_element(input_layer)
+layers.append_element(hidden_layer)
+layers.append_element(output_layer)
+#![add_layers]
+
+#![create_instance]
+NeuralNetwork network(layers)
+network.quick_connect()
+network.initialize_neural_network()
+#![create_instance]
+
+#![set_parameters]
```

Comment:
It only allows for SWIG interfaces, so everything that you can do from python, you can do from meta examples